content/docs/getting-started/technical-documentation/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Adding TechDocs
+lastUpdated: '2021-03-08T21:00:00.0Z'
+description: How to set up Roadie Backstage for TechDocs.
+---
+
+## Introduction
+
+The TechDocs feature of Roadie Backstage allows markdown files written alongside the code of your components to appear in Backstage as styled HTML documentation. Because this documentation is centralized in Backstage, it is more likely to be found and used by other people in your organization.
+
+![a page of basic documentation for a service in the Backstage service catalog](./sample-service-docs-backstage.png)
+
+## Adding documentation to a component
+
+Under the hood, Backstage uses the popular [MkDocs library](https://www.mkdocs.org/) to render documentation.
+
+### Step 1: Add the mkdocs configuration file
+
+Create a file called `mkdocs.yml` in the root of a component you want to document in Backstage. Inside that YAML file, add the following content, replacing `{component-name}` with the human name of your component.
```

Comment:
Maybe add something like 'mkdocs defines the navigation structure of your documentation' or something to explain what this file does

Reply:
I'm going to merge without and add the changes to another PR.